charts/monica/values.yaml

```diff
@@ -0,0 +1,45 @@
+# Default values for PhotoPrism.
+
+image:
+  repository: monica
+  pullPolicy: IfNotPresent
+  tag: 2.19.1-apache
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 80
+
+env:
+  # See the Monica documentation for all available options for environment variables: https://raw.githubusercontent.com/monicahq/monica/master/.env.example
+  APP_ENV: production
```

Comment:
Comment out all these environment variables

Reply:
and add to ct-values.yaml

Reply:
done.